float64 ua
float64 frontier
float64 ease
float64 time
float64 obstacle
float64 map_area
